interview-questions.md

```diff
@@ -8,44 +8,49 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Why would super be used in a Ruby class?
 
-Your answer:
+Your answer: In a Ruby class, super would be used to create a sub class.
 
-Researched answer:
+Researched answer: Super is a keyword that gets called within a parent class and searches for another method with the same name as the parent
 
 2. What is a gem?
 
-Your answer:
+Your answer: A gem in Ruby is a library
 
-Researched answer:
+Researched answer: A gem in Ruby is a package manager that allows you to access methods that were created by other developers
 
 3. What is a relational database? Are there other kinds of databases?
 
-Your answer:
+Your answer: A relational database is a database that is directly related to a particular programming language. There are multiple relational databases.
 
-Researched answer:
+Researched answer: A relational database is stores information. There are tables, rows and columns, each connected in some way.
```

Comment:
Research answers are a great opportunity to dig a little deeper into the concept/subject. Show of your new found knowledge!